pyan/analyzer.py

```diff
@@ -469,52 +544,48 @@ def visit_ImportFrom(self, node):
         # resolve relative imports correctly. The current "here's a set of files,
         # analyze them" approach doesn't cut it.
         #
+        # As a solution, we register imports here and later, when all files have been parsed, resolve them.
+        #
+        # relative imports are currently not supported, i.e. `from ..mod import xy` is not correctly resolved
```

Comment:
This comment out of date? (Relative imports now handled below.)

Reply:
adjusted the text